runner/runners/invoke.go

```diff
@@ -334,6 +339,11 @@ func (inv *Invoker) run(cmd *runner.Command, id runner.RunID, abortCh chan struc
 		stdlog.Write([]byte(header))
 	}
 
+	if inv.diskMonitor != nil {
```

Comment:
Let's create a diskMonitor with no paths that is effectively a no-op on runner construction in place of checking for nil every time we want to use it